src/components/core/DpEditor/DpBoilerPlateModal.vue

```diff
@@ -14,8 +14,9 @@
         @boilerplate-text-added="addBoilerplateText" />
       <div class="flex flex-items-center u-mt">
         <a
-          class="weight--bold font-size-small"
-          :href="Routing.generate('DemosPlan_procedure_boilerplate_list', { procedure: procedureId })">
+            v-if="boilerplateEditViewRoute"
```

Comment:
where does the v-if comes from? When does can be empty?

Reply:
I suggested introducing this by making the `boilerplaeEditViewRoute` prop optional, as I think having the link in the modal should not be mandatory.